### ISSUE TITLE
API Access

### ISSUE THREAD

**Issue by more-by-more** [soundcloud/api#246]
Hey there Soundcloud! We are hoping to get API access for an upcoming project (online radio station). 

We understand applications are closed at the moment but were hoping to get an ETA on when this will become available again? 

Let us know :)

**Comment by ethanschreur** [comment]
same! id love an estimate

**Comment by ethanschreur** [comment]
or a client id and secret that would be even better

**Comment by MariaVWSoundCloud** [comment]
Thank you for your interest in using our APIs - right now we are not planning to reopen the self-serve app credential creation due to a high risk of fraudulent activities, but you can [reach out to us](https://help.soundcloud.com/hc/en-us/requests/new) to get personal access to it. We will ask you for some information about you and your app to make sure it aligns with our Terms of Use.

**Comment by acidclear** [comment]
Hi @MariaVWSoundCloud. Thanks for responding.   I had previously reached out, but they had said they stopped providing API access.  Have things changed?

**Comment by more-by-more** [comment]
Thanks for responding there @MariaVWSoundCloud! We have sent off a submission now via a support ticket. Let us know if you can review that for us!

**Comment by MariaVWSoundCloud** [comment]
I am on it! :) Just (for information for everyone): Tell us your app name, description and how you are going to use our API already in the ticket, so we can shorten the time! 
We have to ask for all these details to make sure, you will comply with our terms of use, thanks for understanding!
Cheers

**Comment by acidclear** [comment]
Thanks so much @MariaVWSoundCloud  much appreciate the efforts on this.

**Comment by MariaVWSoundCloud** [comment]
Great! Have fun! And let us know when we can take a look and play with your app! ;)

**Comment by Drizzt321** [comment]
@MariaVWSoundCloud Can you please look at https://github.com/soundcloud/api/issues/77, this is a LONGSTANDING issue, would LOVE to get the SoundCloud extension usable for all users of Mopidy. It keeps capping out.

**Comment by MariaVWSoundCloud** [comment]
@cloudgreen absolutely - please just create a customer support ticket and we will help you with that. (Link above)

**Comment by slavanomics** [comment]
Hey, is this still the only way forward? I sent a ticket but haven't gotten a reply yet.

**Comment by MariaVWSoundCloud** [comment]
Hi, if you still haven't received a reply, please give me your ticket ID. Best, Maria

**Comment by Mindgamesnl** [comment]
Hey there!

One of my integrations has seemingly disappeared of the face of the earth, which is disrupting some of my users who use the integration fondly. I've opened a ticket about the issue along with proof of how it was accepted, link to the app and a short explanation about usage.

My ticket ID is 2970161

I hope you can help me, @MariaVWSoundCloud, thanks in advance!

Best,
Mats

**Comment by MariaVWSoundCloud** [comment]
Done!

**Comment by sabato-galasso** [comment]
Hey there! We are a non-profit foundation that produces music for many users, we want to develop an app that collects all the multimedia contents uploaded to suoncloud making it easier for our users to listen to music. We have two soundcloud accounts with a paid plan and have requested API app registration via ticket, @MariaVWSoundCloud, hope you can help us. Thanks in advance
Account Ticket 1: 2984619
Account Ticket 2: 2979037

**Comment by haayhappen** [comment]
Hey @MariaVWSoundCloud , could you kindly check ticket number 3009678. Thanks!

**Comment by Kleylux** [comment]
Hello @MariaVWSoundCloud! Hope you are doing well.
Sorry to bother you. I have opened two tickets recently asking for API access, but unfortunately neither of them received a response. Could you please be kind enough to check them? Thank you in advance.
The tickets are 3016331 and 3030605.

**Comment by MariaVWSoundCloud** [comment]
Hi @Kleylux - I replied to the first one and closed the second as duplicate. Your reply to should be routed directly to me, so should hopefully be faster. :)
@cloudgreen: Thank you for your kind words, we are all doing our best (also improving the customer support ;)).

**Comment by Kleylux** [comment]
Thank you very much @MariaVWSoundCloud, for your insanely quick response! I've just replied to the email with the requested information. I hope you get a well-deserved promotion because you're amazing. Cheers!

**Comment by viethung0823** [comment]
Hi @MariaVWSoundCloud hope you doing fine, I have opened a ticket recently but haven't received any reply, can you please take a look at it for me please, the ticket id is "3037897". Have a good day ❤️

**Comment by Beeen** [comment]
Hi @MariaVWSoundCloud 
Thank you so much for helping us all here 🙏
I've opened a ticket on the 17th but still haven't heard back, just checking in case it has fallen through the cracks :)

**Comment by AT2406** [comment]
Hi @MariaVWSoundCloud

My ticket number is 3062401 and I want to try and build a little tool to help my friends and anyone who would find it useful with the syncing of playlists. I've added in the requested details into the ticket but if there's any other details you would need please let me know. Thanks for helping us all out with this!

**Comment by MariaVWSoundCloud** [comment]
@Beeen you were very unlucky, as I just started my vacation that day :) I hope it got solved but if not, please post your ticket ID here.
@AT2406 Ticket was processed.

**Comment by Beeen** [comment]
Hi Maria, 

No worries at all, hope you had a good break! My ticket number is 3113800

Cheers

**Comment by kritzware** [comment]
Hey @MariaVWSoundCloud, I really appreciate your responsiveness on this thread! 

I'm a developer working on behalf of a well-known record label. We're looking to get API access for some features on our website. We've sent a detailed support ticket on how we'd use the API and how we'll follow the terms of service. It also includes some design mockups of our usage to give you some context. Just wanted to leave a comment here so that hopefully this request is looked into soon - thank you very much.

Support ticket: `3123835`

**Comment by teomikitenko** [comment]
> Hi, if you still haven't received a reply, please give me your ticket ID. Best, Maria

Hi. My support ticket 3122622.  I want to try and build a little pet project with soundcloud API

**Comment by Beeen** [comment]
Hi @MariaVWSoundCloud , it looks like my ticket was closed because I missed an email asking for some details. 
I've added the information as a reply to the closure but not sure if you can re-open it?
The chatbot on your website is not replying anymore to let me open a new ticket...could you advise on what to do next?

**Comment by MariaVWSoundCloud** [comment]
@Beeen Your ticket is still open and we sent you a reply on September 24 asking for a few details. Maybe it ended up in your spam email box? We just sent another one. Lmk

**Comment by slavanomics** [comment]
Hi @MariaVWSoundCloud, Are you able to take a look at 3162387? Thank you so much

**Comment by eevee828** [comment]
Hi @MariaVWSoundCloud, I'm so glad to have found this thread! I submitted a support ticket back in May and I have not heard anything since. Not being able to access the API has stopped me developing my app so I would very much appreciate it if you take a look!

The ticket id is 2918124

I'd appreciate it so much if you could take a look!

**Comment by hannah-subtle** [comment]
Hi @MariaVWSoundCloud could you please check ticket id (2569126) 

I need to add my redirect URI please - we are blocked on any development

**Comment by MariaVWSoundCloud** [comment]
@slavanomics Done!
@eevee828 You got a reply.
@hannah-subtle Well, please just reply to the ticket or open a new one, so you can send us the redirect URI.

**Comment by hannah-subtle** [comment]
@MariaVWSoundCloud I have replied twice to request (2569126) has this not come through. I also emailed the support email.

**Comment by eevee828** [comment]
@MariaVWSoundCloud I have no emails showing me a reply, where can I see this ? (Ticket 2918124)

**Comment by MariaVWSoundCloud** [comment]
@eevee828 Maybe the reply ended up in your spam folder. Also: Did you maybe give a wrong or different email address?

**Comment by eevee828** [comment]
@MariaVWSoundCloud I've checked spam and all but haven't found anything. The email was 10meggsc@gmail.com - can the reply be resent ?

**Comment by hannah-subtle** [comment]
@MariaVWSoundCloud did you receive my email this time? 2569126

**Comment by MariaVWSoundCloud** [comment]
@hannah-subtle Thanks - I closed all of the tickets.

**Comment by MariaVWSoundCloud** [comment]
@eevee828 We replied to the new ticket as well.

**Comment by hannah-subtle** [comment]
@MariaVWSoundCloud Thanks Maria I have the redirect now - does this not need to have http or https as I am getting an 
"Oh no!
We can’t find that page. Check the address and try again?"

example of URL - 
https://secure.soundcloud.com/connect?client_id=redacted&response_type=code&state=random_string&redirect_uri=redacted.com/connect

**Comment by RickFr410** [comment]
> Thank you for your interest in using our APIs - right now we are not planning to reopen the self-serve app credential creation due to a high risk of fraudulent activities, but you can [reach out to us](https://help.soundcloud.com/hc/en-us/requests/new) to get personal access to it. We will ask you for some information about you and your app to make sure it aligns with our Terms of Use.

Hello @MariaVWSoundCloud. I need to use the API for a short time for a project, and need to register an app. Is it possible to talk in private to explain my need ?

**Comment by MariaVWSoundCloud** [comment]
Hi, thanks for reaching out - please reach out to us using the support chat
bot and we will look into it. Cheers, Maria


RickFr410 ***@***.***> schrieb am Do. 31. Juli 2025 um 08:31:

**Comment by rootinsomni** [comment]
@MariaVWSoundCloud sorry to reopen but we have an exciting idea for SC.
Ticket number is 3879093, if you could take a look would be great.

Thank you !
